src/System.Devices.Gpio/System.Devices.Gpio/System.Devices.Gpio.csproj

```diff
@@ -1,10 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\..\tools\common.props" />
   <PropertyGroup>
-    <Description></Description>
+    <Description>This library contains usefull APIs to support .NET development for IoT devices.</Description>
+    <Copyright>Microsoft Corporation, All rights reserved.</Copyright>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackageTags>.NET Core GPIO API IoT device</PackageTags>
+    <PackageIconUrl>http://go.microsoft.com/fwlink/?linkid=833199</PackageIconUrl>
```

Comment:
It looks like this is no longer a valid URL. We should remove this attribute from all existing projects and rely on the PackageIconUrl property within the imported common.props:
https://github.com/dotnet/corefxlab/blob/3e2932bdb580538be39cccb9cc1fa306bb41cc02/tools/common.props#L9

Reply:
@ahsonkhan could you submit a PR and fix this directly?  I did the same for my PR feedback earlier.

/cc @edgardozoppi as FYI

Reply:
Yes. Will do.